index.js

```diff
@@ -2651,27 +2651,29 @@
   }
   S.find = def('find', {}, [$.Function, $.Array(a), $Maybe(a)], find);
 
-  //# pluck :: Accessible a => TypeRep b -> String -> Array a -> Array (Maybe b)
+  //# pluck :: Accessible a => String -> Array a -> Array b
   //.
-  //. Takes a [type representative](#type-representatives), a property name,
-  //. and an array of objects and returns an array of equal length. Each
-  //. element of the output array is Just the value of the specified property
-  //. of the corresponding object if the value is of the specified type
-  //. (according to [`is`](#is)); Nothing otherwise.
-  //.
-  //. See also [`get`](#get).
+  //. Takes a property name and an array of objects and returns an array in
+  //. which each element is the value of the specified property of the
+  //. corresponding object. If for some reason the object lacks the specified
+  //. property, a type error is thrown.
```

Comment:
I'd like to leverage the documentation for existing functions:

    Combines [`map`][R.map] and [`prop`](#prop). `pluck(k, xs)` is equivalent
    to `map(prop(k), xs)`.


    [R.map]: http://ramdajs.com/docs/#map